### PR TITLE
Switch to trace extension for tracking dependencies, uncomment dependency correlation code

### PR DESCRIPTION
**Pull request by jpiyali**
Pushing fix for channel that will fix the tests in AISKU in this PR.

### CODE REVIEW

File: vNext/AISKU/Tests/applicationinsights.e2e.tests.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import { ApplicationInsights, IApplicationInsights } from '../src/applicationinsights-web'
 import { Sender } from '@microsoft/applicationinsights-channel-js';
 import { IDependencyTelemetry, ContextTagKeys, Util } from '@microsoft/applicationinsights-common';
+import { TelemetryContext } from '../../extensions/applicationinsights-properties-js/types/TelemetryContext';
```
Comment (markwolff) [inline vNext/AISKU/Tests/applicationinsights.e2e.tests.ts]: import from module

File: vNext/AISKU/src/Initialization.ts [hunk 2]
```diff
@@ -3,11 +3,11 @@
 
 import { IConfiguration, AppInsightsCore, IAppInsightsCore, LoggingSeverity, _InternalMessageId, ITelemetryItem } from "@microsoft/applicationinsights-core-js";
 import { ApplicationInsights } from "@microsoft/applicationinsights-analytics-js";
-import { Util, IConfig, IDependencyTelemetry, IPageViewPerformanceTelemetry,
-         IPageViewTelemetry, IExceptionTelemetry, IAutoExceptionTelemetry, ITraceTelemetry,
-         IMetricTelemetry, IEventTelemetry, IAppInsights, ConfigurationManager } from "@microsoft/applicationinsights-common";
```
Comment (markwolff) [inline vNext/AISKU/src/Initialization.ts]: Was ConfigurationManager unused?

File: vNext/AISKU/Tests/applicationinsights.e2e.tests.ts (hunk 1 above)
Comment (jpiyali) [reply vNext/AISKU/Tests/applicationinsights.e2e.tests.ts]: Fixed

File: vNext/AISKU/src/Initialization.ts (hunk 2 above)
Comment (jpiyali) [reply vNext/AISKU/src/Initialization.ts]: not used

**Pull request merged**